src/php/composer.json

```diff
@@ -4,7 +4,7 @@
   "license": "Apache-2.0",
   "version": "1.73.0",
   "require": {
-    "php": ">=7.0.0",
+    "php": ">=7.1.0",
```

Comment:
Sorry there's some non-obvious about the code base here.

Can you please make the same change in this file as well? https://github.com/grpc/grpc/blob/master/templates/src/php/composer.json.inja#L7

Thanks.